low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -191,12 +191,12 @@ def _raid_health_monitor(self):
                             data = fs.read().rstrip()
                         if self.FAULT_RESOLVED in data:
                             self.alert_type = self.FAULT
-                            self._alert_msg = RaidAlertMsgs.MISMATCH_MSG.value
+                            self._alert_msg = "Inconsistency detected in the RAID disk data, for %s RAID-1 array." %device
```

Comment:
Inconsistencies detected in RAID protected disks, in %s RAID array

Reply:
Done